### PR TITLE
refacotr devops

### PR DESCRIPTION
**Pull request by gaoyan1998**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by leechor — COMMENTED** [review]
LGTM, there are some comments that seem could be removed.

File: dinky-web/src/pages/DevOps/JobDetail/JobHistoryTab.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  });
+
+  const renderVersionList = () => {
+    return (
+      <Row>
+        <Col span={3}>
+          <div id="scrollableDiv">
+            <List
+              size={"small"}
+              header={'版本列表'}
```
Comment (Zzm0809) [inline dinky-web/src/pages/DevOps/JobDetail/JobHistoryTab.tsx]: use i18n

File: dinky-web/src/pages/DevOps/JobDetail/components/logsTab/ExceptionTab.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        const rte = jobDetail?.jobHistory?.exceptions["root-exception"];
+        logs.push({"taskName":"RootException","stacktrace":rte,"exceptionName":rte});
+        logs.push(...jobDetail.jobHistory?.exceptions["exceptionHistory"]["entries"]);
+        return (
+            <Row>
+                <Col span={3}>
+                    <div id="scrollableDiv">
+                        <List
+                            size={"small"}
+                            header={'异常列表'}
```
Comment (Zzm0809) [inline dinky-web/src/pages/DevOps/JobDetail/components/logsTab/ExceptionTab.tsx]: use i18n

File: dinky-web/src/pages/DevOps/JobDetail/components/logsTab/TaskManagerLogsTab.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  }
+
+  const renderLogTab = () => {
+    return (
+      <Row>
+        <Col span={3}>
+          <div id="scrollableDiv">
+            <List
+              size={"small"}
+              header={'TaskManager列表'}
```
Comment (Zzm0809) [inline dinky-web/src/pages/DevOps/JobDetail/components/logsTab/TaskManagerLogsTab.tsx]: use i18n

**Review by Zzm0809 — APPROVED** [review]
LGTM

**Pull request merged**